changelog.txt

```diff
@@ -83,7 +107,7 @@
 - Update: Create task list REST API endpoint #7512
 - Update: Deleted OnboardingEmailMarketing note class #7595
 - Update: Removes the use of the depreciated woocommerce_shared_settings hook. #7480
-- Update: Removes non WooCommerce Admin specific settings from the `wc_admin` namespace in the `wc/data` settings store (ex: countries). #7480
+- Update: Removes non WooCommerce Admin specific settings from the `wc_admin` namespace in the `wc/data` settings store (ex
```

Comment:
Looks like this line got cut off.

Reply:
Thanks, just restored in https://github.com/woocommerce/woocommerce-admin/pull/7991/commits/44ca925eb35fc6fe538c7ed33be0bde8dafddb9e